part-2.py

```diff
@@ -4,13 +4,36 @@
 # the appropriate comment.
 
 # search
+def search(array,query):
```

Comment:
👍 , just be aware that `array[1:]` creates a whole new array with each iteration leading to an O(n^2) time and space complexity.